models/_sector/alm/ethereum/_schema.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+
+models:
+  - name: alm_ethereum_base_trades
```

Comment:
You need the schema for both models here:
- alm_ethereum_base_trades
- arrakis_finance_ethereum_base_trades

Reply:
Fixed.